administrator/components/com_users/tmpl/method/edit.php

```diff
@@ -0,0 +1,163 @@
+<?php
+/**
+ * @package     Joomla.Administrator
+ * @subpackage  com_users
+ *
+ * @copyright   (C) 2022 Open Source Matters, Inc. <https://www.joomla.org>
+ * @license     GNU General Public License version 2 or later; see LICENSE.txt
+ */
+
+// Prevent direct access
+defined('_JEXEC') or die;
+
+use Joomla\Component\Users\Administrator\View\Method\HtmlView;
+use Joomla\CMS\HTML\HTMLHelper;
+use Joomla\CMS\Language\Text;
+use Joomla\CMS\Router\Route;
+
+// phpcs:ignoreFile
+/** @var  HtmlView  $this */
+
+HTMLHelper::_('bootstrap.tooltip', '.hasTooltip');
+
+$cancelURL = Route::_('index.php?option=com_users&task=methods.display&user_id=' . $this->user->id);
+
+if (!empty($this->returnURL))
+{
+	$cancelURL = $this->escape(base64_decode($this->returnURL));
+}
+
+$recordId = (int) $this->record->id ?? 0;
+$method   = $this->record->method ?? $this->getModel()->getState('method');
+$userId   = (int) $this->user->id ?? 0;
+?>
+<div class="card card-body">
+	<form action="<?php echo Route::_(sprintf("index.php?option=com_users&task=method.save&id=%d&method=%s&user_id=%d", $recordId, $method, $userId)) ?>"
+		  class="form form-horizontal" id="com-users-method-edit" method="post">
+		<?php echo HTMLHelper::_('form.token') ?>
+		<?php if (!empty($this->returnURL)): ?>
+		<input type="hidden" name="returnurl" value="<?php echo $this->escape($this->returnURL) ?>">
+		<?php endif; ?>
+
+		<?php if (!empty($this->renderOptions['hidden_data'])): ?>
+		<?php foreach ($this->renderOptions['hidden_data'] as $key => $value): ?>
+		<input type="hidden" name="<?php echo $this->escape($key) ?>" value="<?php echo $this->escape($value) ?>">
+		<?php endforeach; ?>
+		<?php endif; ?>
+
+		<?php if (!empty($this->title)): ?>
+		<?php if (!empty($this->renderOptions['help_url'])): ?>
+		<span class="pull-right float-end">
+			<a href="<?php echo $this->renderOptions['help_url'] ?>"
+			   class="btn btn-sm btn-small btn-default btn-inverse btn-dark"
+			   target="_blank"
+			>
+				<span class="icon icon-question-sign"></span>
```

Comment:
```suggestion
				<span class="icon icon-question-sign" aria-hidden="true"></span>
```

and needs some text - cant be just an icon

Reply:
Yup, I fixed that. I saw it when fixing the other thing you marked up.